diff -Naur a/numpy/core/include/numpy/npy_cpu.h b/numpy/core/include/numpy/npy_cpu.h
--- a/numpy/core/include/numpy/npy_cpu.h	2019-02-21 16:17:07.284072345 +0800
+++ b/numpy/core/include/numpy/npy_cpu.h	2019-02-21 16:03:00.967089392 +0800
@@ -17,6 +17,7 @@
  *              NPY_CPU_SH_BE
  *              NPY_CPU_ARCEL
  *              NPY_CPU_ARCEB
+ *              NPY_CPU_GPTX
  */
 #ifndef _NPY_CPUARCH_H_
 #define _NPY_CPUARCH_H_
@@ -56,6 +57,8 @@
     #define NPY_CPU_S390
 #elif defined(__ia64)
     #define NPY_CPU_IA64
+#elif defined(__gptx__)
+    #define NPY_CPU_GPTX
 #elif defined(__hppa)
     #define NPY_CPU_HPPA
 #elif defined(__alpha__)
diff -Naur a/numpy/core/include/numpy/npy_endian.h b/numpy/core/include/numpy/npy_endian.h
--- a/numpy/core/include/numpy/npy_endian.h	2019-02-21 16:17:07.285072345 +0800
+++ b/numpy/core/include/numpy/npy_endian.h	2019-02-21 15:59:54.813377773 +0800
@@ -46,7 +46,8 @@
             || defined(NPY_CPU_SH_LE)   \
             || defined(NPY_CPU_MIPSEL)  \
             || defined(NPY_CPU_PPC64LE) \
-            || defined(NPY_CPU_ARCEL)
+            || defined(NPY_CPU_ARCEL)   \
+            || defined(NPY_CPU_GPTX)
         #define NPY_BYTE_ORDER NPY_LITTLE_ENDIAN
     #elif defined(NPY_CPU_PPC)          \
             || defined(NPY_CPU_SPARC)   \
